packages/libs/react-components/src/components/ProductIcons/svgs/Pact.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { SVGProps } from 'react';
+
+const Pact: React.FC<SVGProps<SVGSVGElement>> = (
+  props: SVGProps<SVGSVGElement>,
+) => (
+  <svg
+    width={40}
+    height={40}
+    viewBox="0 0 40 40"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <g clipPath="url(#clip0_1324_1306)">
+      <path
+        d="M9.37464 4.21094H19.9294V34.0667C19.9294 34.5134 19.7519 34.9418 19.4361 35.2576C19.1202 35.5735 18.6918 35.7509 18.2452 35.7509H9.37464C8.92796 35.7509 8.49957 35.5735 8.18372 35.2576C7.86787 34.9418 7.69043 34.5134 7.69043 34.0667V5.89515C7.69043 5.44847 7.86787 5.02008 8.18372 4.70423C8.49957 4.38838 8.92796 4.21094 9.37464 4.21094Z"
+        fill="white"
+      />
+      <path
+        d="M9.37464 4.21094H24.8988C27.7872 4.21094 30.5572 5.35833 32.5996 7.4007C34.642 9.44307 35.7894 12.2131 35.7894 15.1015C35.7891 17.9896 34.6416 20.7594 32.5992 22.8015C30.5569 24.8437 27.787 25.9909 24.8988 25.9909H7.69043V5.89515C7.69043 5.44847 7.86787 5.02008 8.18372 4.70423C8.49957 4.38838 8.92796 4.21094 9.37464 4.21094Z"
+        fill="white"
+      />
+      <g style={{ mixBlendMode: 'multiply' }}>
```

Comment:
@TODO: @isaozler blend modes to be replaced with fill-colors

Reply:
ys..will create a new PR for those. 
the story is not done yet.